2023-08-22T00:21:10.7184262Z ##[group]Run actions/setup-python@v4
2023-08-22T00:21:10.7184519Z with:
2023-08-22T00:21:10.7184726Z   python-version: 3.8
2023-08-22T00:21:10.7184963Z   architecture: x64
2023-08-22T00:21:10.7185162Z   cache: pip
2023-08-22T00:21:10.7185360Z   check-latest: false
2023-08-22T00:21:10.7185684Z   token: ***
2023-08-22T00:21:10.7185898Z   update-environment: true
2023-08-22T00:21:10.7186133Z   allow-prereleases: false
2023-08-22T00:21:10.7186330Z ##[endgroup]
2023-08-22T00:21:10.8915106Z ##[group]Installed versions
2023-08-22T00:21:10.9047495Z Successfully set up CPython (3.8.17)
2023-08-22T00:21:10.9051359Z ##[endgroup]
2023-08-22T00:21:11.6997286Z [command]/opt/hostedtoolcache/Python/3.8.17/x64/bin/pip cache dir
2023-08-22T00:21:13.6688321Z /home/runner/.cache/pip
2023-08-22T00:21:14.0369125Z Received 6339595 of 6339595 (100.0%), 40.9 MBs/sec
2023-08-22T00:21:14.0370549Z Cache Size: ~6 MB (6339595 B)
2023-08-22T00:21:14.0448989Z [command]/usr/bin/tar -xf /home/runner/work/_temp/a5ba034c-c551-46ad-82fc-e744621a34e4/cache.tzst -P -C /home/runner/work/GitHub_Stats_A/GitHub_Stats_A --use-compress-program unzstd
2023-08-22T00:21:14.0779273Z Cache restored successfully
2023-08-22T00:21:14.0797511Z Cache restored from key: setup-python-Linux-22.04-Ubuntu-python-3.8.17-pip-60f2a6c0212a9b74a155cd81b58dd07fd8b180833132566260d95f4de42fe218
